samples/bootloader/src/main.c

```diff
@@ -148,25 +149,33 @@ static void boot_from(u32_t *address)
 	}
 
 	__DSB(); /* Force Memory Write before continuing */
-	__ISB(); /* Flush and refill pipeline with updated premissions */
+	__ISB(); /* Flush and refill pipeline with updated permissions */
 
-	VTOR = (u32_t)address;
+	VTOR = fw_info->firmware_address;
```

Comment:
could be that I'm misunderstanding something but couldn't this be possible?
```suggestion
	VTOR = vector_table

Reply:
That's correct, but `fw_info->firmware_address` will read it directly from flash, so I opted for that.